lib/class-factory.js

```diff
@@ -1386,30 +1386,28 @@ dispatcherPrototype = Object.create(Function.prototype, {
   invoke: {
     value (receiver, args) {
       const overloads = this._o;
+      const methodsThatCanBeInvoked = overloads.filter(method => method.canInvokeWith(args));
+      if (methodsThatCanBeInvoked.length === 0) {
+        throwOverloadError(this.methodName, this.overloads, 'argument types do not match any of:');
+      } else if (methodsThatCanBeInvoked.length > 1) {
+        console.warn("Multiple overloads fit the invocation of " + receiver + 
```

Comment:
There's no need to concatenate strings, `console.warn()` will add a space between the stringified value of each argument.